CHANGES.rst

```diff
@@ -24,12 +24,14 @@ Bug fixes
 ^^^^^^^^^
 * Fixed passing ``missing=0`` to ``xclim.core.calendar.convert_calendar``. (:issue:`1562`, :pull:`1563`).
 * Fix wrong `window` attributes in ``xclim.indices.standardized_precipitation_index``, ``xclim.indices.standardized_precipitation_evapotranspiration_index``. (:issue:`1552`  :pull:`1554`).
+* Several spelling mistakes have been corrected within the documentation and codebase. (:pull:`1576`).
 
 Internal changes
 ^^^^^^^^^^^^^^^^
 * The `flake8` configuration has been migrated from `setup.cfg` to `.flake8`; `setup.cfg` has been removed. (:pull:`1569`)
 * The `bump-version.yml` workflow has been adjusted to bump the `patch` version when the last version is determined to have been a `release` version; otherwise, the `build` version is bumped. (:issue:`1557`, :pull:`1569`).
 * The GitHub Workflows now use the `step-security/harden-runner` action to monitor source code, actions, and dependency safety. All workflows now employ more constrained permissions rule sets to prevent security issues. (:pull:`1577`).
+* The `codespell` library is now a development dependency for the `dev` installation recipe with configurations found within `pyproject.toml`. This is also now a linting step and integrated as a `pre-commit` hook. For more information, see the `codespell documentation <https://github.com/codespell-project/codespell>`_ (:pull:`1576`).
 
```

Comment:
Naive/uninformed question: Is the `dev` installation automatically used when git cloning xclim and doing `mamba install -n my_env -f environment.yml`. I think  `xscen` has  `environment_dev.yml`, but not `xclim`, so how does this work here?

Reply:
That's how we've organized `xclim` here. When someone installs `xclim` into a `conda` environment with the `dev` recipe, it pulls the development libraries from PyPI. This hasn't been an issue to date, but if we wanted to mirror the `xscen` approach, adding this file into the codebase is feasible.

Perhaps worth discussing in a separate issue?

Reply:
I don't see a problem with this organization, I just wanted to understand it, to be sure I'm not using a wrong version by error.